templates/sdk-readme.md

```diff
@@ -53,9 +57,13 @@ What is required to build a project using this SDK. Typically a list of prerequi
 
 Detail on what target runtimes are supported by this SDK. May also include detail around what platforms are tested on, though preference should be towards referring those who are interested in testing to consult workflow files directly (e.g. to inspect [strategy matrix configuration](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) for jobs).
 
+## Feature Support
+
+Specify the version of the spec that this SDK targets and link to the [feature support matrix](https://www.ably.io/download/sdk-feature-support-matrix).
+
 ## Known Limitations
 
-A list of features that we do not currently support, but we do plan to add in the future.
+A list of features that we do not currently support, but we do plan to add in the future. If all features are supported, then specify that here.
```

Comment:
I'm a little worried about the added statement as it can quickly fall out of sync with upstream sources of change. I would suggest not having it.

Reply:
Agreed and removed!

Reply:
As a somewhat timely intervention, it's interesting to note that one of our contributors has raised the question as to what to put into this section of a readme if there are no known limitations... see [this internal Slack comment](https://ably-real-time.slack.com/archives/C019LDWM9UH/p1630568852001900?thread_ts=1630563168.001800&cid=C019LDWM9UH).

To speed things up I may add a quick commit myself to the branch underlying this pull request to clarify that (once I've decided what the clarification should be 😆).

Reply:
> what to put into this section of a readme if there are no known limitations

Then the "Known Limitations" section just has a comment that says "be the first to identify a known limitation!"

Reply:
https://github.com/ably/ably-common/pull/102/commits/d495e1be0dbbb8a19151a4e35a27459b8dc8aead